File: app/src/main/java/com/mostafavi/home/food/Data/Food.java
package com.mostafavi.home.food.Data;

/**
 * Created by Reza on 2/2/2018.
 */

public class Food {

    private String name;
    private long dateTime;
    private String image;
    private String description;
    private int like;
    private User user;
    private boolean liked;

    public Food() {
    }

    public Food(String name, long dateTime, String image, String description, int like, User user) {
        this.name = name;
        this.dateTime = dateTime;
        this.image = image;
        this.description = description;
        this.like = like;
        this.user = user;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getDateTime() {
        return dateTime;
    }

    public void setDateTime(long dateTime) {
        this.dateTime = dateTime;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getLike() {
        return like;
    }

    public void setLike(int like) {
        this.like = like;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public boolean isLiked() {
        return liked;
    }

    public void setLiked(boolean liked) {
        this.liked = liked;
    }
}

File: app/src/main/java/com/mostafavi/home/food/Data/Food.kt
package com.mostafavi.home.food.Data

class Food {
    var name: String = ""
    var image: String = ""
    var description: String = ""
    var like: Int = 0
    var dateTime: Long = 0
    var liked: Boolean = false
    var user: User? = null


    constructor()


    constructor(name: String, dateTime: Long, image: String, description: String, like: Int, user: User?) {
        this.name = name
        this.image = image
        this.description = description
        this.like = like
        this.dateTime = dateTime
        this.user = user
    }


}